inputs/simple-list-input.vue

```diff
@@ -97,6 +98,16 @@
       },
       // Add an item to the array
       addItem() {
+        // Only saves values from the autocomplete list if tag creation restriction is enabled
+        if (this.restrictItemCreation) {
+          if (!this.autocompleteValue) {
+            this.autocompleteValue = false;
```

Comment:
If `autocompleteValue` is false and meets the criteria, why do you assign false again to it? 👀